src/renderer/store/AuthStore.ts

```diff
@@ -0,0 +1,20 @@
+import { Authentication } from "@xmcl/user";
+
+import Store from "electron-store";
+
+interface AuthStoreData extends Authentication{
+	loggedIn: true, 
+}
+
+class AuthStore extends Store<AuthStoreData | { loggedIn: false }> {
+	public constructor() {
+		super({
+			name: "auth",
+			defaults: {
+				loggedIn: false
+			}
+		});
+	}
+}
+
+export default new AuthStore();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=5164335)